src/components/AppBar.tsx

```diff
@@ -112,39 +161,48 @@ const _AppBar = ({ location }: RouteComponentProps) => {
       style={{ zIndex: 1 }}
     >
       <NavBarLinks>
-        <Link to={routesEnum.landingPage} className="mx30">
+        <Link to={routesEnum.landingPage} className="mr30">
           <EthLogo src={EthDiamond} alt="eth-diamond" />
-          <div className="flex flex-column center ml5">
-            <BarLinkText
-              active={pathname === routesEnum.landingPage}
-              level={4}
-              margin="none"
-              className="bar-link-text no-padding"
-            >
-              Eth2 Launch Pad
-            </BarLinkText>
-            {!IS_MAINNET && <Text>for {ETH2_NETWORK_NAME} testnet</Text>}
-          </div>
+          {!mobile && (
+            <div className="flex flex-column center ml5">
+              <BarLinkText
+                active={pathname === routesEnum.landingPage}
+                level={4}
+                margin="none"
+                className="bar-link-text no-padding"
+              >
+                <Text>
+                  <FormattedMessage
+                    defaultMessage="Eth2 {network} Launchpad"
+                    values={{
+                      network: IS_MAINNET ? '' : `${TESTNET_LAUNCHPAD_NAME} `,
```

Comment:
The idea here was to show "Eth2 Launchpad" for mainnet (leaving out the word 'mainnet') and to show "Eth2 Pyrmont Launchpad" for Pyrmont testnet.

Reply:
Ya in this case it doesn't matter, `TESTNET_LAUNCHPAD_NAME` & `ETH2_NETWORK_NAME` will both be `Pyrmont`. Having at `ETH2_NETWORK_NAME` is prob a safe bet tho 👍

Reply:
@wackerow 
Sorry I meant replacing `TESTNET_LAUNCHPAD_NAME` with `ETH2_NETWORK_NAME` here. Just felt a bit strange here since we always use `ETH2_NETWORK_NAME` as "where am I". :)

@samajammin
Yeah IMO using `ETH2_NETWORK_NAME` would be more straightforward in this context.